assign_rights/templates/base.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {% include 'head.html' %}
+    {% block extra_css %}{% endblock %}
+  </head>
+  <body>
+    <div class="wrapper">
+
+      {% include 'navbar.html' %}
+
+      <div class="content-wrapper container">
+
+        <!-- Content Header (Page header) -->
```

Comment:
I think it makes sense to move the h1 into `<main>` wrapped in `<div>` instead of `<section>` elements.